dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapActivity.h

```diff
@@ -73,17 +79,45 @@ namespace WindowsAppRuntime::MddBootstrap::Activity
             return m_lastFailure;
         }
 
-        void SetMddBootStrapAPI(MddBootStrapAPI mddBootStrapAPI)
+        void SetMddBootstrapAPI(MddBootstrapAPI mddBootstrapAPI)
+        {
+            m_mddBootstrapAPI = mddBootstrapAPI;
+        }
+
+        void SetInitializeData()
```

Comment:
Odd name as 'set' usually implies 'assign' and 'InitializeData' sounds like there's multiple values being set. But it's just the counter and it's increment not assign.

Rename to `IncrementInitializeCount()` and return the new value

```
uint32_t IncrementInitializeCount()
{
    return ++g_initializationCount;
}
```